src/components/Post/Post.js

```diff
@@ -2,10 +2,26 @@ import React, { useState, useEffect } from 'react';
 import { Route } from 'react-router-dom';
 import PropTypes from 'prop-types';
```

Comment:
We've already imported at the top, but can probably start adding the actual checks for proptypes at the bottom as these components are further built out

Reply:
Good call, we'll want to make sure to go back and add those.